README.md

```diff
@@ -12,7 +12,8 @@ Currently the following testing frameworks are supported:
 | **JavaScript** | Intern, TAP, Karma, Lab, Mocha, Jasmine, Jest         | `intern`, `tap`, `karma`, `lab`, `mocha`, `jasmine`, `jest`       |
 | **Python**     | Nose, Nose2, PyTest, Django, unittest (PyUnit)        | `nose`, `nose2`, `pytest`, `djangotest`, `djangonose`, `pyunit`   |
 | **Elixir**     | ExUnit, ESpec                                         | `exunit`, `espec`                                                 |
-| **Go**         | Go, Ginkgo                                                    | `gotest`, `ginkgo`                                                          |
+| **Erlang**     | CommonTest                                            | `ct`                                                              |
```

Comment:
It should write `commontest` instead of `ct` here. This name refers to the name of this runner in test.vim, which is `CommonTest`, so downcased it becomes `commontest`.